src/skydb.test.js

```diff
@@ -0,0 +1,154 @@
+const axios = require("axios");
+
+const { SkynetClient, genKeyPairFromSeed } = require("../index");
+
+jest.mock("axios");
+
+const dataKey = "testdatakey";
+const data = { example: "This is some example JSON data." };
+const { publicKey, privateKey } = genKeyPairFromSeed("Super Save Test Secret");
+
+const skylink = "AAA_uTgxYiKcqpGMLNe2V52fLc3FivZBZStLVqMSeHnGtQ";
+const client = new SkynetClient("https://siasky.net");
+const dataLink = "sia://AAChv5I6FTTqd8_6mtIOgwd5AhxurcYY9OSc-FacWlurEw";
+const dataLink_dbV2 = "sia://AAA_uTgxYiKcqpGMLNe2V52fLc3FivZBZStLVqMSeHnGtQ";
+const rawEntryData = Uint8Array.from([
+  0, 0, 161, 191, 146, 58, 21, 52, 234, 119, 207, 250, 154, 210, 14, 131, 7, 121, 2, 28, 110, 173, 198, 24, 244, 228,
+  156, 248, 86, 156, 90, 91, 171, 19,
+]);
+const RawBytesData =
+  "[123,34,95,100,97,116,97,34,58,123,34,101,120,97,109,112,108,101,34,58,34,84,104,105,115,32,105,115,32,115,111,109,101,32,101,120,97,109,112,108,101,32,74,83,79,78,32,100,97,116,97,32,50,46,34,125,44,34,95,118,34,58,50,125]";
+
+const RawBytesData_dbV2 =
+  "[123,34,95,100,97,116,97,34,58,123,34,101,120,97,109,112,108,101,34,58,34,84,104,105,115,32,105,115,32,115,111,109,101,32,101,120,97,109,112,108,101,32,74,83,79,78,32,100,97,116,97,46,34,125,44,34,95,118,34,58,50,125]";
+
+jest.setTimeout(60000); // 60 second timeout
+jest.useRealTimers();
+
+beforeEach(() => {
+  axios.mockResolvedValue({ data: { skylink } });
+});
+
+describe("db.setDataLink", () => {
+  it("should be a DataLink set", async () => {
+    await client.db.setDataLink(privateKey, dataKey, dataLink);
+  });
+});
+
+describe("db.setEntryData", () => {
+  it("should set EntryData", async () => {
+    await client.db.setEntryData(privateKey, dataKey, rawEntryData);
+  });
+});
+
+describe("db.getEntryData", () => {
+  it("should get EntryData", async () => {
+    const receivedData = await client.db.getEntryData(publicKey, dataKey);
+    await expect(receivedData["data"]).toEqual(rawEntryData);
+  });
+});
+
+describe("db.getRawBytes", () => {
+  it("should get RawBytes", async () => {
+    const receivedData = await client.db.getRawBytes(publicKey, dataKey);
+    await expect(receivedData["dataLink"]).toEqual(dataLink);
+    await expect("[" + receivedData["data"] + "]").toEqual(RawBytesData);
+  });
+});
+
+describe("db.setJSON", () => {
+  it("should set jsonData to skydb", async () => {
+    const receivedData = await client.db.setJSON(privateKey, dataKey, data);
+
+    await expect(receivedData.dataLink).toEqual(`sia://${skylink}`);
+  });
+});
+
+describe("db.getJSON", () => {
+  it("should get jsonData from skydb", async () => {
+    const receivedData = await client.db.getJSON(publicKey, dataKey);
+
+    await expect(receivedData.data).toEqual(data);
+  });
+});
+
+describe("db.deleteJSON", () => {
+  it("should delete jsonData on skydb", async () => {
```

Comment:
Please add a test that verifies that the `jsonData` has been deleted.

Reply:
Yes, I just saw it. 
I add one more "db.getJSON" with "await expect(receivedData.data).toEqual(null);" for confirming that verifies that the jsonData has been deleted.